Desktop/modules/dynamicmodule.cpp

```diff
@@ -476,6 +477,7 @@ std::string DynamicModule::getLibPathsToUse()
 	for(const std::string & path : requiredLibPaths)
 		libPathsToUse += ", '" + path + "'";
 
+	libPathsToUse += ", '" + AppDirs::programDir().absolutePath().toStdString() + "/../R/library" + "'";
```

Comment:
Good catch, considering our discussion about the build structure, this kind of makes sense on Linux at Build since binaries are in `build/Desktop/JASP` and so they are in `../R/library` if it's local, but it's probably going to fail on the Flatpak build. 

I'm going to tweak things a bit, and we see how Flatpak goes, and we can continue debugging from there.

Reply:
@JorisGoosen, this should just work here, right?

```
libPathsToUse += ", '" + AppDirs::rHome() + "/library" + "'";
```

Reply:
Yeah that looks good, and I will run into it anyway when Ill rebase `enginePerModule` or `development` (if it is in there already by then).